tests/static_files/test_file_serving_resolution.py

```diff
@@ -14,6 +16,8 @@
 if TYPE_CHECKING:
     from litestar.types import FileSystemProtocol
 
+mimetypes.encodings_map[".br"] = "br"
```

Comment:
I think this needs to be done somewhere in library code, not the tests - as we need this to work if someone is running their app on 3.8.

Reply:
You're right, I didn't think that through